saiplayer/saiplayer.cpp

```diff
@@ -1273,6 +1309,8 @@ void printUsage()
     std::cout << "        Enable temporary view between init and apply" << std::endl << std::endl;
     std::cout << "    -i --inspectAsic:" << std::endl;
     std::cout << "        Inspect ASIC by ASIC DB" << std::endl << std::endl;
+    std::cout << "    -s --sleep:" << std::endl;
+    std::cout << "        Sleep after success reply, to notice fdb notifications" << std::endl << std::endl;
```

Comment:
is this only for fdb notification, could be any notification, e.g. port oper status notification?

Reply:
actually no, you will see all notifications since it will just sleep and sairedis will receive all

Reply:
changed to any